proposals/0003-external-calls.md

```diff
@@ -0,0 +1,344 @@
+# Introduce External Calls
+
+* Proposal: [FIP-0003](0003-external-calls.md)
+* Author: [Daniel Hails](https://github.com/djrhails) & [Alexander Harkness](https://github.com/bearbin) 
+* Review Manager:  [Susan Eisenbach](https://github.com/SusanEisenbach)
+* Status: **Awaiting review**
+* Issue label: [0003-external-calls](https://github.com/franklinsch/flint/issues?q=is%3Aopen+is%3Aissue+label%3A0003-external-calls)
+
+## Introduction
+
+Contracts can be created "from outside" via Ethereum transactions or from within Flint Contracts. They contain persistent data in state variables and functions that can modify these variables. Calling a function on a different contract (instance) will perform an EVM Function call and thus switch the context such that state variables in the old context are inaccessible.
+
+## Motivation
+Calls to untrusted contracts can introduce several unexpected risks and errors. External calls may execute malicious code in that contract _or any other contract_ that it depends upon. As such, **every** external call should be treated as a security risk.
```

Comment:
This paragraph could do with redrafting to make it slightly clearer/flow better.